dd-sdk-android/src/main/kotlin/com/datadog/android/rum/webview/RumWebViewClient.kt

```diff
@@ -27,6 +27,7 @@ import com.datadog.android.rum.RumResourceKind
  * This will map the page loading, and webview errors into RUM Resource and
  * Error events respectively.
  */
+@Deprecated("You should use the DatadogEventBridge JavaScript interface instead.")
```

Comment:
Maybe we should add a snippet or indication here on _how to use_ the `DatadogEventBridge`.